Source/NNSupportFunctions/arm_nn_vec_mat_mul_result_acc_s8_s16.c

```diff
@@ -42,75 +40,85 @@
 /*
  *  Refer to header file for details.
  */
-void arm_nn_vec_mat_mul_result_acc_s8(const int8_t *lhs_in,
-                                      const int8_t *rhs_in,
-                                      const int32_t *bias,
-                                      int16_t *dst,
-                                      const int32_t dst_offset,
-                                      const int32_t dst_multiplier,
-                                      const int32_t dst_shift,
-                                      const int32_t rhs_cols,
-                                      const int32_t rhs_rows,
-                                      const int32_t batch)
+arm_cmsis_nn_status arm_nn_vec_mat_mul_result_acc_s8_s16(const int8_t *lhs,
+                                                         const int8_t *rhs,
+                                                         const int32_t *effective_bias,
+                                                         const int32_t *bias,
+                                                         int16_t *dst,
+                                                         const int32_t lhs_offset,
+                                                         const int32_t dst_offset,
+                                                         const int32_t dst_multiplier,
+                                                         const int32_t dst_shift,
+                                                         const int32_t rhs_cols,
+                                                         const int32_t rhs_rows,
+                                                         const int32_t activation_min,
+                                                         const int32_t activation_max,
+                                                         const int32_t batches,
+                                                         const int32_t batch_offset)
 {
-    for (int i_batch = 0; i_batch < batch; ++i_batch)
+    (void)dst_offset;
```

Comment:
A lot of unused here. Let's remove these completely. Since it is a support function we could add it back relatively easy if needed.